### DISCUSSION TITLE
.border strange behavior

### DISCUSSION THREAD

**Discussion by abdennourzahaf** [tailwindlabs/tailwindcss#5610] [Help] [answered]
I applied `.border` trying to get `border-width: 1px;`, but I ended up getting `border: 1px solid #dee2e6!important;`.
Since It's using `!important`, it's impossible to change `border-color` while keeping 1px width.

This is from chrome dev tools.
![image](https://user-images.githubusercontent.com/62243290/134822661-58d6a280-b5e4-4038-9cdb-9931bef1e41d.png)

**Comment by adamwathan** [comment, marked as answer]
Hey! Looks like that style comes from Bootstrap, not Tailwind.

https://stackoverflow.com/questions/49479632/why-is-there-an-important-override-on-the-border-classess-color-property-in-boo/49480060
